gui/scripts/syncthing/core/views/directives/majorUpgradeModalView.html

```diff
@@ -0,0 +1,26 @@
+<div id="majorUpgrade" class="modal fade" tabindex="-1" data-backdrop="true" data-keyboard="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header alert alert-danger">
+        <h4 class="modal-title">
+          <span ng-if="icon" class="glyphicon glyphicon-chevron-up"></span>
+          <span translate>Major Upgrade</span>
+        </h4>
+      </div>
+      <div class="modal-body">
+        <p>
+          <span translate>This is a major version upgrade.</span>
```

Comment:
This requires a companion change to the `translate()` function in `build.go` and possibly `cmd/translate/main.go` to use multiple inputs -- that stuff generates the `lang-en.json` translation source from the translate-tagged tags in the HTML source.